src/NuGet.Core/NuGet.CommandLine.XPlat/Strings.resx

```diff
@@ -456,4 +456,15 @@ For more information, visit http://docs.nuget.org/docs/reference/command-line-re
     <comment>{0} - Operation Name
 {1} - project file path</comment>
   </data>
+  <data name="Error_MultipleMatchingSpecs" xml:space="preserve">
+    <value>Multiple matching package specs found when attempting to add package '{0}' to project '{1}'. Number of matching package specs found are  '{2}'.</value>
```

Comment:
This is a good debug message, but it isn't very actionable for the user. I would avoid calling it a package spec here, just call it project instead. It might be helpful to say that this command only supports adding to a single project, but multiple projects with the same path were found.

A user should really never hit this, it is more of an internal nuget issue. But I like the idea of having an error message for these scenarios like you have here.